src/resolver/CompanyRepository.ts

```diff
@@ -0,0 +1,17 @@
+import {Db} from "mongodb";
+import {Company} from "./Company";
+
+export class CompanyRepository {
+    public static async doesCompanyExist(db: Db, id: string): Promise<Boolean> {
+        return null != await db.collection('User').findOne({_id: id});
+    }
+
+    public static async getCompanyById(db: Db, _id: string): Promise<{ exists: Boolean, company: Company }> {
```

Comment:
Hier in diesem Fall ist ein Rückgabewert von `null` ausreichend um zu signalisieren, dass die Company nicht existiert.